src/coders/sequencer-batch.ts

```diff
@@ -1,5 +1,5 @@
 import { add0x, remove0x, encodeHex } from '../common'
-import { Contract, BigNumber } from 'ethers'
+import { Contract, BigNumber, ethers } from 'ethers'
```

Comment:
We should import `utils` here instead of `ethers` and then just use `utils` directly

Reply:
I see. This change makes me think that I should've merged this PR and #8. 

Hopefully this isn't too much of a headache, but I'll rename this PR again and include the changes from #8. 

I'll also re-pull the `data-transport-layer` to include anything from the changes you merged in https://github.com/ethereum-optimism/data-transport-layer/pull/40

Reply:
On second thought, going to keep 8 since it resolves a separate issue. Mistook your suggestion as targeting the utils of this utility library